android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/backstack/BackStackManager.kt

```diff
@@ -77,7 +78,8 @@ internal class BackStackManager<C : Parcelable>(
         class Pop<C : Parcelable> : Wish<C>()
         class SaveInstanceState<C : Parcelable> : Wish<C>()
         class ShrinkToBundles<C : Parcelable> : Wish<C>()
-        class TearDown<C : Parcelable> : Wish<C>()
+        class TearDownRouting<C : Parcelable> : Wish<C>()
+        class ReinitRouting<C : Parcelable> : Wish<C>()
```

Comment:
Can we clarify those wishes?
Maybe I am not getting something, but for me, it looks similar (without digging the code):
`TearDownRouting`
`ReinitRouting`
`NewRoot`

Reply:
I already removed them from this class, will push soon.

Reply:
Done.